check-pull-request.sh

```diff
@@ -99,9 +105,9 @@ EOL
 }
 
 delete_comment() {
-  pushd "$DIR/github-comment" >> /dev/null
+  pushd "$GITHUB_COMMENT_SRC_DIR" >> /dev/null
   # No recommended changes, so delete any existing comment
-  swift run github-comment \
+  echo swift run github-comment \
```

Comment:
This doesn't actually run anything, does it?

Reply:
Looks like the travis command line tool auto-formatted the file. I'm going to pull the secure variables generation out to a separate PR.

Reply:
I mean the `echo` command here. It looks like you're disabling this command?  Am I misreading the use of `echo`?

Reply:
Removed! Good catch.